### ISSUE TITLE
Fetching elements never ends

### ISSUE THREAD

**Issue by slavek-kucera** [eclipse/che-che4z-explorer-for-endevor#183]
After upgrading from 0.13.2 to 0.14.0 I am no longer able to list elements.
The extension is stuck in the "Fetching elements" state.
The location I am trying to access contains 5 elements (3 sourced, 2 no-sources).

**Comment by VitGottwald** [comment]
Just ran into this when helping another developer. There is about 25k elements in the response from Endevor and it takes about 10s for the data to come back. But it takes over another minute for the Endevor Explorer to process that. It got my CPU fans to start spinning.

**Comment by nickImbirev** [comment]
Small update:
After playing with the extension with big numbers of Endevor elements, I can say, that the infinite searching behaviour probably is not because of E4E, but because of Endevor SDK. I even tried to search for elements for the same Endevor location (with more than 100 thousands of elements) using Zowe CLI Endevor plugin and I saw the same infinite search behaviour there.

I cannot say, why this behaviour started in the 0.14.0 version of E4E precisely, but we also upgraded the version of the Endevor SDK in this update, maybe it is somehow related to it.

I will try to go deeply into the problem, but it seems like we will need some help from the Endevor SDK team.
